apollo-assembly/src/main/resources/application-github.properties

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2023 Apollo Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# DataSource
+spring.datasource.url=jdbc:h2:mem:~/apolloassemblydb;mode=mysql;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;BUILTIN_ALIAS_OVERRIDE=TRUE;DATABASE_TO_UPPER=FALSE
+#spring.datasource.username=
+#spring.datasource.password=
+
+# Table prefix
+spring.jpa.hibernate.naming.physical-strategy=com.ctrip.framework.apollo.common.jpa.TablePrefixNamingStrategy
+spring.jpa.table-prefix.config-prefix=C_0_
+spring.jpa.table-prefix.portal-prefix=P_0_
+spring.session.jdbc.table-name=P_0_SPRING_SESSION
+
+# H2 datasource
+spring.sql.init.schema-locations=classpath:jpa/@@platform@@/apolloconfigdb.sql, classpath:jpa/@@platform@@/apolloportaldb.sql
```

Comment:
I observed that four new SQL files have been added: h2/apolloconfigdb.sql, h2/apolloportaldb.sql, mysql/apolloconfigdb.sql, and mysql/apolloportaldb.sql. Were these files generated automatically or edited manually? Additionally, is there a simpler method to manage them?

Reply:
By the way, I observed two new SQL files in the `scripts/sql/assembly` directory. Are these identical to the SQL files located in the `jpa/mysql` folder?